WKData/Sources/WKData/Data Controllers/WKGrowthTasks/WKGrowthTasksDataController.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+
+public final class WKGrowthTasksDataController {
+
+    var service = WKDataEnvironment.current.mediaWikiService
+    let project: WKProject
+
+    public init (project: WKProject) {
+        self.project = project
+    }
+
+    // MARK: GET Methods
+
+    // TODO: to get more tasks, send used IDS to `ggtexcludepageids` parameter
+    // TODO: remove debug prints
+    public func getGrowthAPITask(task: WKGrowthTaskType = .imageRecommendation, completion: @escaping (Result<[WKGrowthTask.Page], Error>) -> Void) {
+
+        guard let service else {
+            return
+        }
+        var pages: [WKGrowthTask.Page] = []
+
+        let parameters = [ "action": "query",
+                           "generator": "growthtasks",
+                           "formatversion": "2",
+                           "format": "json",
+                           "ggttasktypes": "\(task.rawValue)",
+                           "ggtlimit": "10"
+        ]
+
+        guard let url = URL.mediaWikiAPIURL(project: project) else {
+            return
+        }
+
+        let request = WKMediaWikiServiceRequest(url: url, method: .GET, parameters: parameters)
+        service.performDecodableGET(request: request) { (result: Result<WKGrowthTaskAPIResponse, Error>) in
+
+            switch result {
+            case .success(let response):
+                print(response)
+                pages.append(contentsOf: self.getTaskPages(from: response))
+                completion(.success(pages))
+            case .failure(let error):
+                print(error)
```

Comment:
Wasn't sure if you were going to tackle failures later but for now you can pass along the error here:

`completion(.failure(error))`